app/scripts/controllers/transactions/index.js

```diff
@@ -1200,10 +1200,12 @@ export default class TransactionController extends EventEmitter {
           txMeta.chainId,
         );
 
-        const quoteVsExecutionRatio = `${new BigNumber(tokensReceived, 10)
-          .div(txMeta.swapMetaData.token_to_amount, 10)
-          .times(100)
-          .round(2)}%`;
```

Comment:
Why is this change needed?

Reply:
Great question! For some reason I was getting empty `txMeta.txReceipt.logs` on Rinkeby: 
![image](https://user-images.githubusercontent.com/80175477/127540536-d4bedf2d-b3df-479c-84f2-912ee77a3870.png)

When that happens, the `getSwapsTokensReceivedFromTxMeta` fn call returns null (https://github.com/MetaMask/metamask-extension/blob/develop/ui/pages/swaps/swaps.util.js#L707). Any idea why there are on logs on Rinkeby?